sample-kotlin/src/main/java/com/fblitho/lithoktsample/errors/TitleComponentSpec.kt

```diff
@@ -27,10 +28,10 @@ object TitleComponentSpec {
 
   @OnCreateLayout
   fun onCreateLayout(c: ComponentContext, @Prop(resType = STRING) title: String): Component =
-      Text.create(c)
-          .text(title)
-          .textStyle(BOLD)
-          .textSizeDip(24f)
-          .positionDip(YogaEdge.BOTTOM, 16f)
-          .build()
+      text(c) {
```

Comment:
I get a little stuck on the two `text` functions following each other like this. They add a little confusion albeit not the end of the world.

I'm also struggling to find a good replacement. 

I guess one way could be to do something like: (which would require some changes to the extension function)
```
text(c, title) {
  .textStyle(BOLD)
  .textSizeDip(24f)	        
  .textSizeDip(24f)
  .positionDip(YogaEdge.BOTTOM, 16f)	        positionDip(YogaEdge.BOTTOM, 16f)
}
```